packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeErrorRenderer.tsx

```diff
@@ -3,32 +3,42 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
+import { ReactElement } from "react";
 import { Anchor, unstable_ErrorRegion as ErrorRegion, Text } from "@itwin/itwinui-react/bricks";
 import { MAX_LIMIT_OVERRIDE } from "../internal/Utils.js";
 import { HierarchyLevelDetails, useTree } from "../UseTree.js";
 import { ErrorNode } from "./FlatTreeNode.js";
 import { useLocalizationContext } from "./LocalizationContext.js";
 
 /** @alpha */
-export interface TreeErrorItemProps {
+interface TreeErrorItemProps {
   /** A callback to reload a hierarchy level when an error occurs and `retry` button is clicked. */
   reloadTree?: (options: { parentNodeId: string | undefined; state: "reset" }) => void;
   /** Action to perform when the filter button is clicked for this node. */
   onFilterClick?: (hierarchyLevelDetails: HierarchyLevelDetails) => void;
 }
-
+/** @alpha */
 interface TreeErrorRendererOwnProps {
+  /** List of errors to be displayed */
   errorList: ErrorNode[];
+  // Callback to render custom error messages. Component should be wrapped in `ErrorRegion.Item` from `@itwin/itwinui-react` package.
+  renderError?: ({ errorNode, scrollToElement }: { errorNode: ErrorNode } & Pick<LinkedNodeProps, "scrollToElement">) => ReactElement;
```

Comment:
couldn't `{ errorNode: ErrorNode } & Pick<LinkedNodeProps, "scrollToElement">` be replaced with just `LinkedNodeProps`?